kfs-core/src/main/java/edu/arizona/kfs/fp/document/authorization/GeneralErrorCorrectionCorrectingAccountingLinesAuthorizer.java

```diff
@@ -42,11 +42,18 @@ public boolean isGroupEditable(AccountingDocument accountingDocument, List<? ext
 
     @Override
     public boolean hasEditPermissionOnAccountingLine(AccountingDocument accountingDocument, AccountingLine accountingLine, String accountingLineCollectionProperty, Person currentUser, boolean pageIsEditable) {
-        return false;
+        if (accountingLine.getSequenceNumber() == null) {
+            return false;
+        }
+        boolean retval = super.hasEditPermissionOnAccountingLine(accountingDocument, accountingLine, accountingLineCollectionProperty, currentUser, pageIsEditable);
+        return retval;

```

Comment:
Could this be refactored to work more like the DistributionOfIncomeAndExpenseAccountingLineAuthorizer maybe the determineEditPermissiononField() and determineEditPermissionOnLine() since both of these classes are extending the same class?